templates/block/block--menu-block--news-topics.html.twig

```diff
@@ -44,13 +44,13 @@
 {% set attributes = attributes.setAttribute('id', heading_id) %}
 {% set attributes = attributes.addClass(classes) %}
 
-<div{{ attributes }}>
+<nav{{ attributes }} aria-labelledby="button-label">
```

Comment:
Attributes should be added to the `{{ attributes }}` object. See: https://www.drupal.org/docs/8/theming-drupal-8/using-attributes-in-templates

Code Example:

```
{% set attributes = attributes.setAttribute('aria-labelledby', 'button-label') %}
```